libs/blocks/text/text.css

```diff
@@ -108,7 +108,79 @@
   padding-left: var(--spacing-xs);;
 }
 
+/* Legal */
+.legal.text-block .foreground {
+  padding: var(--spacing-xl) 0;
+}
+
+.legal.text-block p {
+  font-style: italic;
+  font-size: var(--type-body-xxs-size);
+  line-height: var(--type-body-xxs-lh);
+}
+
+/* link-farm */
+.link-farm.text-block {
+  padding: var(--spacing-xl) 0;
+}
+
+.link-farm.text-block .foreground {
+  display: grid;
+  grid-template-columns: auto;
+  gap: var(--spacing-xl) var(--spacing-m);
+}
+
+.link-farm.text-block h3 {
+  margin: 0 0 var(--spacing-l) 0;
+  font-size: var(--type-heading-xs-size);
+  line-height: var(--type-heading-xs-lh);
+}
+
+.link-farm.text-block h2 {
+  margin: 0 0 var(--spacing-m) 0;
+}
+
+.link-farm.text-block p {
+  font-size: var(--type-body-s-size);
+  margin: 0 0 var(--spacing-s) 0;
+  line-height: var(--type-body-s-lh);
+}
+
+.link-farm.text-block p:last-child {
+  margin: 0;
+}
+
+.link-farm.text-block a {
+  color: inherit;
+  text-decoration: underline;
+}
+
+.link-farm.text-block a:hover {
+  text-decoration: double underline;
+}
+
+/* Desktop */
+@media screen and (min-width: 1200px) {
+  .link-farm.text-block h2 {
+    margin: 0 0 var(--spacing-xl-static) 0;
+  }
+
+  .link-farm.text-block .foreground {
+    grid-template-columns: repeat(4, auto);
+  }
+}
+
 /* Tablet */
+@media screen and (min-width:600px) and (max-width: 1200px) {
+  .link-farm.text-block h2 {
```

Comment:
This is declared twice in 600 < screen < 1200 and screen > 1200. Please move it to screen > 600

Reply:
Resolved this!

Reply:
I don't think it's resolved; before you had other declarations for 600 < screen < 1200, now they disappeared. My suggestion was to only move the `.link-farm.text-block h2` part to `min-width: 600` since it was declared twice.
If you think you don't need separate styles for 600 < screen < 1200, I'm fine, but your initial PR had them

Reply:
Also, if you only keep screen < 600, maybe, you can add in the existing media query https://github.com/adobecom/milo/pull/778/files#diff-d8c01efdf4985f2e26b4ddb2e913847c4140966396259eae7797f24a58e2a55aR173

Reply:
@drashti1712 - please check the feedback provided ^

Reply:
Hi @narcis-radu, sorry for the confusion. I have pushed the code with the changes according to your comments. Please review.